3_Implementation/documentation/html/search/all_b.js

```diff
@@ -0,0 +1,457 @@
+var searchData=
+[
+  ['teardown_29',['tearDown',['../_t_i_c-_t_a_c-_t_o_e__unittest_8c.html#ab804e0f66def63262c1d6b4e3d2a2bd7',1,'tearDown():&#160;TIC-TAC-TOE_unittest.c'],['../unity_8h.html#a9909011e5fea0c018842eec4d93d0662',1,'tearDown(void):&#160;TIC-TAC-TOE_unittest.c']]],
+  ['test_5fabort_30',['TEST_ABORT',['../unity__internals_8h.html#a35ff53d9ad3c4ec9d611fdf04319a23b',1,'unity_internals.h']]],
+  ['test_5fassert_31',['TEST_ASSERT',['../unity_8h.html#ae0a72bc174ac97e90c4a711e5e52d540',1,'unity.h']]],
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/2015pushkar/C-mini-Project-303459-LTTS/pullRequest?prid=7744834)
```suggestion
  ["test_5fassert_31",['TEST_ASSERT',['../unity_8h.html#ae0a72bc174ac97e90c4a711e5e52d540',1,'unity.h']]],
```